AK/Format.cpp

```diff
@@ -652,8 +652,18 @@ void vout(FILE* file, StringView fmtstr, TypeErasedFormatParams params, bool new
 }
 #endif
 
+static bool is_debug_enabled = true;
+
+void set_debug_enabled(bool value)
+{
+    is_debug_enabled = value;
+}
```

Comment:
How many of these disabling methods do we have with this? 3?

Reply:
Well, `DebugLogStream::set_enabled` will soon go, what other functions are there?

Reply:
I think we also had an environment variable, although thinking on it, that might've been for `dbg()`

Reply:
Yes, we had that in `test-js`, I updated that:

https://github.com/asynts/serenity/blob/bf33690bb3d286200a0669f31c5e3e7fb3649b17/Userland/Utilities/test-js.cpp#L730-L732